testsuite/src/it/java/org/graalvm/tests/integration/AppReproducersTest.java

```diff
@@ -924,6 +925,113 @@ public void jdkReflections(TestInfo testInfo, Apps app) throws IOException, Inte
         }
     }
 
+    @Test
+    @Tag("builder-image")
+    @IfMandrelVersion(min = "23.1.8", max = "23.1.999", inContainer = true)
+    public void vthreadsPropsContainer23_1Test(TestInfo testInfo) throws IOException, InterruptedException {
+        vthreadsProps(testInfo, Apps.VTHREADS_PROPS_BUILDER_IMAGE);
+    }
+
+    @Test
+    @IfMandrelVersion(min = "23.1.8", max = "23.1.999")
+    public void vthreadsProps23_1Test(TestInfo testInfo) throws IOException, InterruptedException {
+        vthreadsProps(testInfo, Apps.VTHREADS_PROPS);
+    }
```

Comment:
The fix is not yet backported to 23.1, I would prefer to enable this only after the fact.

Reply:
It's mentioned here: graalvm-community-jdk21u/issues/66 and I was about to do the backport. Wrote a test first.

Reply:
I know, I am just saying that I would like to have the test enabled only after doing the backport :)

I suggest commenting it out for the time being.

Reply:
@zakkak Commented out in [00ba907](https://github.com/Karm/mandrel-integration-tests/pull/335/commits/00ba907c0ad4e008a1aa7d9501b024d098bc0dec)